examples/gallery/layout/plot_with_columns.ipynb

```diff
@@ -36,15 +36,9 @@
     "        x, y, color=color, by=facet, subplots=True, padding=0.1, width=None, height=None, cmap=cmap).opts(\n",
     "        'Scatter', min_height=300, responsive=True, min_width=200)\n",
     "\n",
-    "pn.Column(\n",
-    "    '### Auto MPG Explorer',\n",
-    "    plot,\n",
-    "    pn.Row(\n",
-    "        pn.WidgetBox(x, y, color),\n",
-    "        pn.WidgetBox(facet)\n",
-    "    ),\n",
-    "    width_policy='max'\n",
-    ").servable()"
+    "pn.Column('### Auto MPG Explorer', plot,\n",
+    "    pn.Row(pn.WidgetBox(x, y, color), pn.WidgetBox(facet)),\n",
+    "    width_policy='max').servable()"
```

Comment:
I understand you don't like the previous formatting, although it is basically the de-facto standard since the introduction and wide adoption of [black](https://github.com/python/black) in the Python community, but I have to say that I really dislike this formatting too.

Reply:
Here's what black produces when given this bit of code:

```python
pn.Column(
    "### Auto MPG Explorer",
    plot,
    pn.Row(pn.WidgetBox(x, y, color), pn.WidgetBox(facet)),
    width_policy="max",
).servable()
```

Reply:
What black produces is fine, but the old way was very messy to my eyes.  Feel free to revise it to black's format.